Add `Language Surveillance Division` (#6059)
ppy/osu-wiki: wiki/People/The_Team/Language_Surveillance_Division/en.md

---
tags:
  - chatop
  - chat op
  - chat operator
  - chatops
  - lsd
---

# Language Surveillance Division

*For the modern team of moderators, see: [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team).*

> **Language Surveillance Division** (colloquially *LSD*) was a usergroup responsible for the moderation of language-specific communities inside osu!. Each of its members had a dedicated area of focus: a language channel where they were mainly present, and respective forum section that they could manage.<!-- TicClick: no to-do, as it's hard enough to accomplish, but having the early history of the team (its roots, and a complete list of all chat moderators, would be appreciated -->

Reviewer: ```suggestion
**Language Surveillance Division** (colloquially *LSD*) was a usergroup responsible for the moderation of language-specific communities inside osu!. Each of its members had a dedicated area of focus: a language channel where they were mainly present, and respective forum sections that they could manage.<!-- TicClick: no to-do, as it's hard enough to accomplish, but having the early history of the team (its roots, and a complete list of all chat moderators, would be appreciated -->
```

> **Language Surveillance Division** (colloquially *LSD*) was a usergroup responsible for the moderation of language-specific communities inside osu!. Each of its members had a dedicated area of focus: a language channel where they were mainly present, and respective forum sections that they could manage.<!-- TicClick: no to-do, as it's hard enough to accomplish, but having the early history of the team and its roots, and a complete list of all chat moderators, would be appreciated -->

Reviewer 2: ```suggestion
**Language Surveillance Division** (colloquially known as the *LSD*) was a usergroup responsible for the moderation of language-specific communities inside osu!. Each of its members had a dedicated area of focus: a language channel where they were mainly present, and respective forum sections that they could manage (such as places like [Help](https://osu.ppy.sh/community/forums/5) or [Off-Topic](https://osu.ppy.sh/community/forums/52)). <!-- TicClick: no to-do, as it's hard enough to accomplish, but having the early history of the team and its roots, and a complete list of all chat moderators, would be appreciated -->
```

## Tasks

> Members of LSD were hand-picked by the osu!team from the community to act as moderators and be examples of good behaviour. Due to this fact, they didn't have any noticeable chat or forum features, such as colored nickname, or a custom forum title. On the [IRC](/wiki/Internet_Relay_Chat), however, they were [operators](/wiki/Internet_Relay_Chat#why-are-some-usernames-prefixed-with-different-signs?) and could apply [silences](/wiki/Silence).

Reviewer 2: ```suggestion
Members of LSD were hand-picked by the osu!team from the community to act as moderators for lower traffic communities and subforums. Unlike Global Moderators, they did not have features like colored nicknames, or a custom forum title. On [IRC](/wiki/Internet_Relay_Chat), however, they were [operators](/wiki/Internet_Relay_Chat#why-are-some-usernames-prefixed-with-different-signs?) and could apply [silences](/wiki/Silence).
```

Reviewer: hyphen needed

```suggestion
Members of LSD were hand-picked by the osu!team from the community to act as moderators for lower-traffic communities and subforums. Unlike Global Moderators, they did not have features like coloured nicknames, or custom forum titles. On [IRC](/wiki/Internet_Relay_Chat), however, they were [operators](/wiki/Internet_Relay_Chat#why-are-some-usernames-prefixed-with-different-signs?) and could apply [silences](/wiki/Silence).
```

> The LSD also served as an entry point to general community overseeing: chat moderators who showed interest and had been deemed capable of more difficult and large-scale management tasks were promoted to the [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team).<!-- internal reference: https://osu.ppy.sh/community/forums/topics/123465 -->

Reviewer 2: ```suggestion
The LSD also served as an intermediary group for promising members pending admission into the [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) at times, as the two groups generally shared a common goal and similar skill sets.<!-- internal reference: https://osu.ppy.sh/community/forums/topics/123465 -->
```

> ## Reformation

Reviewer 2: ```suggestion
## Restructuring
```

{{{ Infobox
![](img/chat-moderator-profile.png "Profile page of a chat moderator in 2013")
}}}

> After the [Staff Restructuring](https://osu.ppy.sh/community/forums/topics/123510) in 2013, Language Surveillance Division had been renamed to the **Chat Moderators** and received its own members listing<!-- https://web.archive.org/web/20140209160121/http://osu.ppy.sh:80/g/18 -->. The usergroup had also become home for moderators who were only present in large subforums, such as [Help](https://osu.ppy.sh/community/forums/5) or [Off-Topic](https://osu.ppy.sh/community/forums/52).

Reviewer: ```suggestion
After a [staff restructuring](https://osu.ppy.sh/community/forums/topics/123510) in 2013, the Language Surveillance Division had been renamed to the **Chat Moderators** and received its own members listing<!-- https://web.archive.org/web/20140209160121/http://osu.ppy.sh:80/g/18 -->. The usergroup had also become home for moderators who were only present in large subforums, such as [Help](https://osu.ppy.sh/community/forums/5) or [Off-Topic](https://osu.ppy.sh/community/forums/52).
```

> After a [staff restructuring](https://osu.ppy.sh/community/forums/topics/123510) in 2013, the Language Surveillance Division had been renamed to the **Chat Moderators** and received its own members listing<!-- https://web.archive.org/web/20140209160121/http://osu.ppy.sh:80/g/18 -->. The usergroup had also become home for moderators who were only present in large subforums, such as [Help](https://osu.ppy.sh/community/forums/5) or [Off-Topic](https://osu.ppy.sh/community/forums/52).

Reviewer 2: ```suggestion
After a [staff restructuring](https://osu.ppy.sh/community/forums/topics/123510) in 2013, the Language Surveillance Division was renamed as the **Chat Moderators** and received its own members listing<!-- https://web.archive.org/web/20140209160121/http://osu.ppy.sh:80/g/18 -->.
```

> On [August 20, 2015](/wiki/Staff_Log/2015#august), the Chat Moderators and Global Moderation teams [were merged](https://osu.ppy.sh/community/forums/posts/4435972), effectively giving all moderators distinctive red nicknames in the chat and management power over the whole forum. 

Reviewer: ```suggestion
On [August 20, 2015](/wiki/Staff_Log/2015#august), the Chat Moderators and Global Moderation teams [were merged](https://osu.ppy.sh/community/forums/posts/4435972), effectively giving all moderators distinctive red nicknames in chat and management power over the whole forum. 
```
